tests/toktx-tests.cmake

```diff
@@ -238,8 +238,8 @@ gencmpktx( astc_ldr_10x5_FlightHelmet_baseColor   astc_ldr_10x5_FlightHelmet_bas
 gencmpktx( astc_ldr_8x8_FlightHelmet_baseColor    astc_ldr_8x8_FlightHelmet_baseColor.ktx2   ../srcimages/FlightHelmet_baseColor.png "--test --encode astc --astc_blk_d 8x8" "" "")
 gencmpktx( astc_ldr_12x10_FlightHelmet_baseColor  astc_ldr_12x10_FlightHelmet_baseColor.ktx2 ../srcimages/FlightHelmet_baseColor.png "--test --encode astc --astc_blk_d 12x10" "" "")
 gencmpktx( astc_ldr_12x12_FlightHelmet_baseColor  astc_ldr_12x12_FlightHelmet_baseColor.ktx2 ../srcimages/FlightHelmet_baseColor.png "--test --encode astc --astc_blk_d 12x12" "" "")
-gencmpktx( astc_ldr_6x6_Iron_Bars_001_normal      astc_ldr_6x6_Iron_Bars_001_normal.ktx2     ../srcimages/Iron_Bars/Iron_Bars_001_normal.jpg "--test --assign_oetf linear --normal_mode --encode astc --astc_blk_d 6x6" "" "")
-gencmpktx( astc_ldr_5x4_Iron_Bars_001_normal      astc_ldr_5x4_Iron_Bars_001_normal.ktx2     ../srcimages/Iron_Bars/Iron_Bars_001_normal.jpg "--test --assign_oetf linear --normal_mode --encode astc --astc_blk_d 5x4" "" "")
+gencmpktx( astc_ldr_6x6_Iron_Bars_001_normal      astc_ldr_6x6_Iron_Bars_001_normal.ktx2     ../srcimages/Iron_Bars/Iron_Bars_001_normal_unnormalized.png "--test --assign_oetf linear --normal_mode --encode astc --astc_blk_d 6x6" "" "")
+gencmpktx( astc_ldr_5x4_Iron_Bars_001_normal      astc_ldr_5x4_Iron_Bars_001_normal.ktx2     ../srcimages/Iron_Bars/Iron_Bars_001_normal_unnormalized.png "--test --assign_oetf linear --normal_mode --encode astc --astc_blk_d 5x4" "" "")
```

Comment:
The renaming of the file is missing.

Reply:
I mean renaming of the file in the srcimages folder or adding the file to that folder.

Reply:
It shows up for me in the changes. <img width="1612" alt="Screenshot 2021-12-23 at 10 26 58" src="https://user-images.githubusercontent.com/17299876/147227105-2f5b2ee2-8b77-4805-bc33-f05d9cc72b94.png">

Reply:
Ahh! sorry. Missed that too due to the name being truncated on the left to fit the narrow width GitHub uses.